### PR TITLE
Code review 2021.11.09

### PR DESCRIPTION
**Pull request by andrewrgoss**
Pep8 formatting, fix typos, and removed unused modules

### CODE REVIEW

File: coindecko/pyscript/ingest_data.py [hunk 1]
```diff
@@ -1,49 +1,47 @@
...
     # Code here will react to failed requests
     else:
-        print('There is a failed request! ', response.status_code)   
-    # Decode the JSON response into a dictionary and use the data
+        print('There is a failed request! ', response.status_code)
+        # Decode the JSON response into a dictionary and use the data
     data = response.json()
     # Debug by printing out the first data extracted
-    try:
-        print(data[0])
```
Comment (andrewrgoss) [inline coindecko/pyscript/ingest_data.py:26]: Better to use a breakpoint in a debugger here with the print statement than a try-catch -- a print statement function is not going to throw an exception

File: coindecko/pyscript/ingest_data.py [hunk 2]
```diff
@@ -1,49 +1,47 @@
...
+
+def test_data_ingested(df):
+    if df[0]['id'] == 'bitcoin':
+        df = pd.DataFrame(df, columns=['id', 'name', 'current_price', 'total_volume', 'market_cap', 'last_updated'])
+        return df
+    else:
+        print('Wrong data imported! Kindly check the data source')
+
+
 while True:
```
Comment (andrewrgoss) [inline coindecko/pyscript/ingest_data.py:20]: The test_data_ingested() function doesn't return a boolean value? I think that's what you're trying for here but we can further review this piece.

**Review by denironyx — APPROVED** [review]
Thanks for the changes. I'm approving the PR.

File: coindecko/pyscript/ingest_data.py (hunk 1 above)
Comment (denironyx) [reply coindecko/pyscript/ingest_data.py:26]: Do you mean that red dot at the side of the editor? I never understood how that works. Maybe, you could show me.

File: coindecko/pyscript/ingest_data.py (hunk 2 above)
Comment (denironyx) [reply coindecko/pyscript/ingest_data.py:20]: Sure, we can further review it.

**Pull request merged**